src/api.js

```diff
@@ -17,6 +17,17 @@ export const getArticleById = (article_id) =>{
     })
 }
 
+export const patchArticleVotes = (article_id, votesToPatch) => {
+    const voteUpdate = { inc_votes: votesToPatch };
+    return newsApi.patch(`/articles/${article_id}`, voteUpdate)
+    .then(function (response) {
+        console.log(response);
```

Comment:
pls remove console log

Reply:
There is a sentence: Click to login for voting and comments: right where to click. I thought that would be enough...